XmlParser/src/ua/goit/alg/test/XMLParserTest.java

```diff
@@ -0,0 +1,51 @@
+package ua.goit.alg.test;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Test;
+import ua.goit.alg.parser.Handler;
+import ua.goit.alg.parser.Tag;
+import ua.goit.alg.parser.XmlParser;
+import ua.goit.alg.parser.XmlParserImpl;
+
+public class XMLParserTest {
+  private final static String xml = "<?><note w=\"fff\" m=\"fff\">text</note>";
+  private XmlParser parser = new XmlParserImpl();
+  @Test
+  public void parserTest() throws IOException {
```

Comment:
I'm ok with tests which are not passing (I know XML parsing is hard and boring) - you can put @Ignore on the tests. Just show me a) you know the problems with the thing you've created b) you know when to stop writing the tests